Sources/UIExtensions/Components/SwiftUI/Pure/EnumeratedList.swift

```diff
@@ -10,6 +10,7 @@ public struct EnumeratedList<TextView: View>: View {
     let verticalSpacing: CGFloat
     let circledNumberStrokeColor: Color
     let textView: (String) -> TextView
+    let enumerationTextView: (String) -> TextView
```

Comment:
you might want to have a new generic for that because now we expect textView and enumerationTextView will have same type (same modifiers applied)

Reply:
also maybe view builder annotations, I am not really sure if they help with the performance

Reply:
Good point there, took care of it